### ISSUE TITLE
Consider migrating tide status context to Tide

### ISSUE THREAD

**Issue by spiffxp** [kubernetes/test-infra#9324]
From slack @cblecker 

> because the Submit Queue [context] was capitalized, it alpha sorted to the top of the context list
> which makes a difference in k/k when you have a bazillion statuses
> this is not the case with tide

/area prow/tide
/kind cleanup
/sig testing

**Comment by spiffxp** [comment]
FYI @cjwagner do we think it would be feasible to make this change this release cycle? Might still be nice.

**Comment by cjwagner** [comment]
Yeah, if we made the `tide` status context name configurable we could do that this release cycle. Changing the context name requires a status migration so we shouldn't impose that on all currently existing `tide` instances.
